### DISCUSSION TITLE
Regarding Confusions

### DISCUSSION THREAD

**Discussion by udhay111** [tnodir/fort#43] [Q&A] [answered]
# Little silly Dumb clarification needed
see through the screen shot below I am little bit confused
![Image 1](https://user-images.githubusercontent.com/76847070/203567088-443f7893-3208-45eb-ba5e-cdaa55c7c741.png)
- Application's Internet is blocked if it is set to application group enabled state and working as it intended to be.
- But I am confused why options below is not greyed out use application group enabled state is set 
- Is the intended purpose is missing or I am dumb please explain this feauture a little bit as i donot know the intended purpose.

**Comment by tnodir** [comment, marked as answer]
1. _Allow_/_Block_ rules of Programs window always **override** such rules of App Groups.
So, the result of the following will be _allowed_:
    - _allowed_ `avp.exe` in Programs;
    -  _blocked_ `avp.exe` in App Groups.

2. Obviously, "Use App Group's enabled state" is for enabling/disabling the Program by App Group.

3. _Allow_/_Block_ rule of Program can be edited independently of enabled state.
